### PR TITLE
[MM-28363] Add asset for user limit overage emails

### PR DESCRIPTION
**Pull request by nickmisasi**
#### Summary
This asset is needed for an upcoming PR around user limit overage emails. 

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-28363?filter=15333

#### Related Pull Requests
N/A (yet)

#### Screenshots

image

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -290,6 +290,7 @@ var config = {
                 {from: 'images/warning.png', to: 'images'},
                 {from: 'images/logo-email.png', to: 'images'},
                 {from: 'images/browser-icons', to: 'images/browser-icons'},
+                {from: 'images/cloud', to: 'images'},
```
Comment (marianunez) [inline webpack.config.js:293]: Should this be limited to the specific png we want?

Comment (devinbinnie) [inline webpack.config.js:293]: Does this break any of the other images we're using?

**Review by devinbinnie — APPROVED** [review]

File: webpack.config.js (hunk 1 above)
Comment (nickmisasi) [reply webpack.config.js:293]: Thinking here was that any future assets could be added without messing with webpack. I don't think there's any significant negative impact, but I'm happy to change to be explicit - let me know what you think!

@devinbinnie (will continue your discussion here) This shouldn't break anything, simply marks it in webpack to be servedby URL /static/images/<png>

Comment (devinbinnie) [reply webpack.config.js:293]: Okay, as long as there's no explicit reference to the images (eg. in CSS files)

**Review by marianunez — APPROVED** [review]

File: webpack.config.js (hunk 1 above)
Comment (nickmisasi) [reply webpack.config.js:293]: Since those are included via imports, they should be fine. I will be sure to give one last check before merging.

Comment (nickmisasi) [reply webpack.config.js:293]: Confirmed, good to go!

**Pull request merged**